x-pack/legacy/plugins/siem/public/components/page/network/network_top_countries_table/columns.tsx

```diff
@@ -0,0 +1,176 @@
+import { IS_OPERATOR } from '../../../timeline/data_providers/data_provider';
+import { Provider } from '../../../timeline/data_providers/provider';
+import * as i18n from './translations';
+import { PreferenceFormattedBytes } from '../../../formatted_bytes';
+
+export type NetworkTopCountriesColumns = [
+  Columns<NetworkTopCountriesEdges>,
+  Columns<TopNFlowNetworkEcsField['bytes_in']>,
+  Columns<TopNFlowNetworkEcsField['bytes_out']>,
+  Columns<NetworkTopCountriesEdges>,
+  Columns<NetworkTopCountriesEdges>,
+  Columns<NetworkTopCountriesEdges>
+];
+
+export type NetworkTopCountriesColumnsIpDetails = [
+  Columns<NetworkTopCountriesEdges>,
+  Columns<TopNFlowNetworkEcsField['bytes_in']>,
+  Columns<TopNFlowNetworkEcsField['bytes_out']>,
+  Columns<NetworkTopCountriesEdges>,
+  Columns<NetworkTopCountriesEdges>
+];
+
+export const getNetworkTopCountriesColumns = (
+  indexPattern: StaticIndexPattern,
+  flowTarget: FlowTargetSourceDest,
+  type: networkModel.NetworkType,
+  tableId: string
+): NetworkTopCountriesColumns => [
+  {
+    name: i18n.COUNTRY,
+    render: ({ node }) => {
+      const geo = get(`${flowTarget}.country`, node);
+      const geoAttr = `${flowTarget}.geo.country_iso_code`;
+      const id = escapeDataProviderId(`${tableId}-table-${flowTarget}-country-${geo}`);
+      if (geo != null) {
+        return (
+          <DraggableWrapper
+            key={id}
+            dataProvider={{
+              and: [],
+              enabled: true,
+              id,
+              name: geo,
+              excluded: false,
+              kqlQuery: '',
+              queryMatch: { field: geoAttr, value: geo, operator: IS_OPERATOR },
+            }}
+            render={(dataProvider, _, snapshot) =>
+              snapshot.isDragging ? (
+                <DragEffects>
+                  <Provider dataProvider={dataProvider} />
+                </DragEffects>
+              ) : (
+                <>
+                  <CountryFlagAndName countryCode={geo} />
+                </>
+              )
+            }
+          />
+        );
+      } else {
+        return getEmptyTagValue();
+      }
+    },
+    width: '20%',
+  },
+  {
+    align: 'right',
+    field: 'node.network.bytes_in',
+    name: i18n.BYTES_IN,
+    sortable: true,
+    render: bytes => {
+      if (bytes != null) {
+        return <PreferenceFormattedBytes value={bytes} />;
+      } else {
+        return getEmptyTagValue();
+      }
+    },
+  },
+  {
+    align: 'right',
+    field: 'node.network.bytes_out',
+    name: i18n.BYTES_OUT,
+    sortable: true,
+    render: bytes => {
+      if (bytes != null) {
+        return <PreferenceFormattedBytes value={bytes} />;
+      } else {
+        return getEmptyTagValue();
+      }
+    },
+  },
+  {
+    align: 'right',
+    field: `node.${flowTarget}.flows`,
+    name: i18n.FLOWS,
+    sortable: true,
+    render: flows => {
+      if (flows != null) {
+        return numeral(flows).format('0,000');
```

Comment:
No change necessary since we're not doing it anywhere else, but I wonder if we should be using the `format:number:defaultPattern` Kibana advanced setting to format these numerals?